Podfile

```diff
@@ -2,7 +2,12 @@ platform :ios, '9.0'
 # ignore all warnings from all pods
 inhibit_all_warnings!
 target 'ooniprobe' do
-    pod 'mkall', :git => 'https://github.com/measurement-kit/mkall-ios.git'
+  pod 'measurement_kit',
+#    :git => 'https://github.com/measurement-kit/mkall-ios.git'
+    :git => 'https://github.com/measurement-kit/ios-libs.git'
+#    pod 'measurement_kit',
+#        :git => 'https://github.com/measurement-kit/measurement-kit.git',
+#	:tag => 'v0.8.3'
```

Comment:
@lorenzoPrimi for example, this feels like wrong and we _don't want_ to include it.